### PR TITLE
[barracuda] Upgrade ECS to 8.0.0

### PR DESCRIPTION
**Pull request by r00tu53r**
Automated PR.

Upgrades ECS to 8.0.0.

### CODE REVIEW

File: packages/barracuda/data_stream/waf/fields/ecs.yml [hunk 1]
```diff
@@ -153,7 +151,7 @@
 - external: ecs
   name: process.pid
 - external: ecs
-  name: process.ppid
+  name: process.parent.pid
```
Comment (efd6) [inline packages/barracuda/data_stream/waf/fields/ecs.yml:154]: Breaking change?

File: packages/barracuda/manifest.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 format_version: 1.0.0
 name: barracuda
 title: Barracuda Logs
-version: 0.7.1
+version: 0.7.2
```
Comment (efd6) [inline packages/barracuda/manifest.yml:4]: This is arguably 0.8.0 given the breaking change in parent PID mapping.

Comment (r00tu53r) [reply packages/barracuda/manifest.yml:4]: Yes! Scripting error. I'll fix this. Thanks for catching it :)

Comment (r00tu53r) [reply packages/barracuda/manifest.yml:4]: All rsa2elk integrations (including this one) will have another round of modifications after updates.

**Comment by adriansr** [comment]
Replaced by #2574

**Pull request closed without merging**